src/Shell/BakeShell.php

```diff
@@ -73,6 +73,13 @@ public function startup()
         if (isset($this->params['connection'])) {
             $this->connection = $this->params['connection'];
         }
+
+        if ($this->params['quiet']) {
+            $this->interactive = false;
+            if (isset($this->{$task}) && !in_array($task, ['Project'])) {
```

Comment:
Why are you using `in_array()` with a single element? Couldn't that check be done with `$task !== 'Project'`

Reply:
Copy and paste from the similar code above.
I thought this must have a reason (extendability) to be this way there.